src/main.module.ts

```diff
@@ -24,13 +26,11 @@ import { LoggerModule } from 'nestjs-pino';
         },
       },
     }),
-    MsSqlMdmDatabaseModule,
-    MsSqlCedarDatabaseModule,
-    MsSqlCisDatabaseModule,
-    MsSqlNumberGeneratorDatabaseModule,
+    AuthModule,
+    DatabaseModule,
     MdmModule,
   ],
   controllers: [],
-  providers: [],
+  providers: [AuthService],
```

Comment:
Out of curiousity, do we need to declare `AuthService` as a provider here? Is that fact that `AuthModule` declares it as a provider not enough?

Reply:
good point - we don't need it here

Reply:
this is updated now